org.eclipse.jdt.ls.product/syntaxServer.product

```diff
@@ -37,6 +39,10 @@
       <plugin id="org.eclipse.core.runtime"/>
       <plugin id="org.eclipse.core.variables"/>
       <plugin id="org.eclipse.debug.core"/>
+      <plugin id="org.eclipse.e4.core.contexts"/>
+      <plugin id="org.eclipse.e4.core.di"/>
+      <plugin id="org.eclipse.e4.core.di.annotations"/>
+      <plugin id="org.eclipse.e4.core.services"/>
```

Comment:
Have a test on lightweight and standard mode. Without `"javax.annotation, "org.eclipse.e4.core.contexts, org.eclipse.e4.core.di, org.eclipse.e4.core.di.annotations, org.eclipse.e4.core.services"`, both modes can start successfully. Looks like these 5 plugins are not required.

Reply:
You are right. Fixed.